backend/src/main/java/se/chalmers/dat265/group1/api/order/OrderController.java

```diff
@@ -36,34 +42,43 @@ public Order getOrder(String orderID) {
     }
 
     @Override
-    public Order createNewOrder(Order order) {
+    public Order createNewOrder(Order order) throws InvalidDateFormatException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'se.chalmers.dat265.group1.api.order.InvalidDateFormatException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)